app/controllers/admins_controller.rb

```diff
@@ -2,8 +2,8 @@ class AdminsController < ApplicationController
     before_action :authenticate_user!
     
     def index
-      @items=Item.all
-      @procedures=Procedure.all
+      @items=Item.where("status = 'pending'")
+      @procedures=Procedure.where("status = 'pending'")
       @user=current_user
```

Comment:
you don't need to set the variable @user here because you can just refer to current_user from the view.

Reply:
Deleted @user in the controller and used current_user in the form.